### PR TITLE
Improve chart interaction when scrolling is on/off

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: testing/tests/DevExpress.viz.core/translator2D.tests.js [hunk 1]
```diff
@@ -2237,6 +2238,160 @@ QUnit.test('All translators process special cases', function(assert) {
...
 QUnit.module("Zooming and scrolling");
 
+QUnit.test('scroll. Scrolling page at right side (right edge of the chart, event not canceled)', function(assert) {
+    var range = $.extend({ minVisible: 70, maxVisible: 99.5, invert: false }, numericRange),
+        canvas = $.extend({}, canvasTemplate),
+        translator;
+
+    translator = new translator2DModule.Translator2D(range, canvas, { isHorizontal: true, breaksSize: 0 });
+
+    assert.ok(translator);
```
Comment (roman-simionov) [inline testing/tests/DevExpress.viz.core/translator2D.tests.js]: I think this assertion is unnecessary

**Review by roman-simionov — DISMISSED** [review]

File: js/events/pointer.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 "use strict";
 
-var support = require("../core/utils/support"),
+var jQuery = require("jquery"),
```
Comment (nightskylark) [inline js/events/pointer.js]: We should not require jquery outside the jquery integration folder. I think these changes should be moved there.

Comment (nightskylark) [reply js/events/pointer.js]: Or maybe you need to configure `special` via registerEvent if you want to patch noJQuery strategy too.

**Review by nightskylark — CHANGES_REQUESTED** [review]

File: js/events/pointer.js [hunk 3]
```diff
@@ -95,6 +96,20 @@ each(EventStrategy.map, function(pointerEvent, originalEvents) {
     registerEvent(pointerEvent, new EventStrategy(pointerEvent, originalEvents));
 });
 
+if(jQuery && support.touch) {
+    each([TouchStrategy.map["dxpointerdown"], TouchStrategy.map["dxpointermove"]], function(_, eventName) {
+        jQuery.event.special[eventName] = {
+            setup: function(_, namespaces, handler) {
+                if(namespaces.indexOf(TouchStrategy.POINTER_EVENTS_NAMESPACE) > -1) {
+                    this.addEventListener(eventName, handler, { passive: false });
```
Comment (nightskylark) [inline js/events/pointer.js]: And, please, use `domAdapter.listen` instead of `addEventListener`

**Review by nightskylark — CHANGES_REQUESTED** [review]

File: js/events/pointer.js (hunk 3 above)
Comment (dxBeardedBear) [reply js/events/pointer.js]: fixed

File: js/events/pointer.js (hunk 2 above)
Comment (dxBeardedBear) [reply js/events/pointer.js]: fixed

File: testing/tests/DevExpress.viz.core/translator2D.tests.js (hunk 1 above)
Comment (dxBeardedBear) [reply testing/tests/DevExpress.viz.core/translator2D.tests.js]: fixed

**Review by nightskylark — APPROVED** [review]

**Pull request merged**